catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -208,4 +222,48 @@ public void setFeatureService(FeatureService featureService) {
   public void setEndpointUtil(EndpointUtil util) {
     this.util = util;
   }
+
+  private Map<String, Object> getLatLonLiteral(String query) {
+    String queryStringNumbersOnly = query.replaceAll("[^\\.\\-\\d]", " ");
+    if (queryStringNumbersOnly.isEmpty()) {
+      return null;
+    }
+    String[] numbersAsStrings = queryStringNumbersOnly.trim().split(" ");
+    if (numbersAsStrings.length % 2 != 0) {
+      Arrays.asList(numbersAsStrings).remove(numbersAsStrings.length - 1);
+      if (numbersAsStrings.length == 0) {
+        return null;
+      }
+    }
+    ArrayList<Double> numbers =
```

Comment:
Left side of a variable declaration should be the highest abstraction possible. This should be `List<Double> numbers`